shell/common/engine_unittests.cc

```diff
@@ -19,6 +22,19 @@ namespace flutter {
 
 namespace {
 
+using ::testing::Invoke;
+using ::testing::ReturnRef;
+
+static void PostSync(const fml::RefPtr<fml::TaskRunner>& task_runner,
```

Comment:
This is in an unnamed namespace, you can omit the `static` here:

```suggestion
void PostSync(const fml::RefPtr<fml::TaskRunner>& task_runner,
```